hummingbot/market/binance/binance_market.pyx

```diff
@@ -974,6 +978,10 @@ cdef class BinanceMarket(MarketBase):
             order_decimal_amount = f"{decimal_amount:f}"
             if order_type is OrderType.LIMIT:
                 decimal_price = self.c_quantize_order_price(symbol, price)
+                notional_size = decimal_amount * decimal_price
```

Comment:
On second thought... I think this is not the right approach here - even though I suggested this solution yesterday.

The problem has to do with how the quantize functions are normally used. e.g. in arbitrage strategy. The strategy checks whether it can expect to send a certain order to both markets, before it sends it. If you simply error one of the orders out like this, then it'll cause the user to be exposed.

We should allow these checks to proceed as intended without raising errors. So I guess the correct solution is to change the interface of `c_quantize_order_amount()` to accept an optional price keyword argument. This means the interface in market base will need to be changed and all market classes will need to have it.

Reply:
okays, will update this